lost/src/main/java/com/mapzen/android/lost/internal/FusedLocationProviderApiImpl.java

```diff
@@ -62,7 +69,10 @@ public void removeLocationUpdates(LocationListener listener) {
 
     @Override
     public void removeLocationUpdates(PendingIntent callbackIntent) {
-        throw new RuntimeException("Sorry, not yet implemented");
+        intentToRequest.remove(callbackIntent);
+        if (intentToRequest.isEmpty()) {
```

Comment:
Good catch. Although probably we want to check if `listenerToRequest` and `intentToRequest` are both empty rather than null, right? Updated in 8c0c5c1276f1fde195b71e543e64aa2a1218b794.